src/main/java/uk/gov/justice/hmpps/prison/api/model/AssessmentQuestion.java

```diff
@@ -23,6 +24,9 @@
     @ApiModelProperty(value = "Question", position = 1)
     private String question;
 
-    @ApiModelProperty(value = "Answer", position = 2)
+    @ApiModelProperty(value = "The answer given. More than one answer might have been given, in which case the other answers will be in the additionalAnswers property", position = 2)
     private String answer;
+
+    @ApiModelProperty(value = "If a question has more than one answer, all but the first answer will be in this property", position = 3)
+    private List<String> additionalAnswers;
```

Comment:
are there any integration test changes that should be made for this?

Reply:
We tend to use integration tests to cover the happy path(s) and deal with details (like this) in the service layer unit tests